src/app/shell/header/header-default/header-default.component.html

```diff
@@ -73,7 +73,9 @@
       </div>
     </div>
     <div class="col-md-5 text-right">
-      <ish-lazy-header-quickorder></ish-lazy-header-quickorder>
+      <div class="d-none d-md-block" *ishFeature="'quickorder'">
```

Comment:
It is nice to have a consistent pattern in regards to applying directives and CSS classes etc. And since the conditional directive is mostly more important it is better to read if it is in first place (see the other changes).

`<div *ishFeature="'quickorder'" class="d-none d-md-block">`

It reads much cleaner "render only if quickorder feature is enabled" and only in this case the CSS classes are relevant.